src/api/bannerAdmin.ts

```diff
@@ -28,7 +28,7 @@ export const updateBanner = async ({
     Bugsnag.notify(e, (event) => {
       event.addMetadata("context", {
         location: "api/bannerAdmin::updateBanner",
-        venueId,
+        venueId: spaceId,
```

Comment:
Changed it a bit in https://github.com/sparkletown/sparkle/pull/2952/commits/38d6ac6a94dd7aa2549bf7fcb0ad9d2fc7a05a13, but didn't completely remove it because it still contains some useful for debugging info